public_asset/lib/data/datasource/public_asset_datasource.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/core/session/session.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/jmap/mail/extensions/public_asset/public_asset.dart';
+
+abstract class PublicAssetDatasource {
+  const PublicAssetDatasource();
```

Comment:
IMO, we do not need it

Reply:
It's only to make the the class implementing it `const` too, and `const` class is better for memory.